src/state/transactions/updater.tsx

```diff
@@ -24,7 +26,20 @@ export function toSerializableReceipt(receipt: TransactionReceipt): Serializable
   }
 }
 
+function getElapsedTime(): number | undefined {
+  // Only log this metric for the first swap of a session.
```

Comment:
This should be a doc-comment on the function, so it is obvious that the function is only meant to return elapsed time from the first time it is called.